EveripediaNetwork/ep-ui
# Fix biome setup

# Summary

Closes https://github.com/EveripediaNetwork/issues/issues/2989

Fixes the issue where biome wasn't identifying linting issues consistently, and biome ignore rules weren't working. Also fixes the issue of linting issues being too much to fix at once. It does so by 
- updating the biome version
- updating biome config file to remove deprecated keys
- configuring biome to lint and check only staged changes locally on and changed files on ci

## How should this be tested?
- create a branch off this one
- make some changes
- try running `pnpm lint` to test that only staged files are linted locally, and only files changed in the branch are linted on ci
- push to github, see that ci linting works fine
